script.js

```diff
@@ -0,0 +1,135 @@
+const speakersObject = {
+  speakerOne: {
+    picture: 'images/face1-b.jpg',
+    name: 'John Smith',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerTwo: {
+    picture: 'images/face2-b.jpg',
+    name: 'Jane Apple',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerThree: {
+    picture: 'images/face3-b.jpg',
+    name: 'Fred Prodsis',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerFour: {
+    picture: 'images/face4-b.jpg',
+    name: 'Megane Froths',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerFive: {
+    picture: 'images/face5-b.jpg',
+    name: 'Ossan Droth',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+  speakerSix: {
+    picture: 'images/face6-b.jpg',
+    name: 'Roths Mirde',
+    bio: 'Professor at Harvard',
+    info: 'John studies web-develpment and published his book The Web Of The World in 2012',
+  },
+};
+
+const speakerArray = Object.keys(speakersObject);
+
+const speakersDiv = document.getElementById('speakers-div');
+const speakersHeader = document.createElement('h2');
+const speakershr = document.createElement('hr');
+const speakersFlex = document.createElement('div');
+const showMore = document.createElement('button');
+
+speakersDiv.classList.add(
+  'd-flex',
+  'flex-column',
+  'align-items-center',
+  'bg-white',
+  'w-100',
+  'py-md-5',
+);
+speakersHeader.classList.add('fw-bold', 'mt-5');
+speakershr.classList.add('horizontal-line', 'color-2', 'm-0', 'mb-md-4');
+speakersFlex.classList.add(
+  'd-flex',
+  'flex-wrap',
+  'gap-5',
+  'container',
+  'align-items-center',
+  'justify-content-center',
+  'my-5',
+);
+showMore.classList.add('show-more-button', 'd-md-none');
+
+function displaySpeakers() {
+  document.querySelector('.d-md-flex').classList.remove('d-none', 'd-md-flex');
+  if (document.querySelector('.d-md-flex') === null) {
+    showMore.onclick = '';
+    showMore.textContent = 'End Of List';
+  }
+}
+
+speakersHeader.textContent = 'Featured Speakers';
+showMore.innerHTML = 'More <span class="color-2">&#9661;</span>';
+
+speakersDiv.append(speakersHeader, speakershr, speakersFlex, showMore);
+
+let i = 0;
+speakerArray.forEach((speaker) => {
```

Comment:
An easier way to do this:
```suggestion
speakerArray.forEach((speaker, i) => {
	console.log(i); // Prints the index at which the loop is currently at
});
```

Reply:
Thanks, it looks cleaner like this